packages/app-store/applecalendar/index.ts

```diff
@@ -9,7 +9,7 @@ export const metadata = {
   installed: true,
   type: "apple_calendar",
   title: "Apple Calendar",
-  imageSrc: "apps/apple-calendar.svg",
+  imageSrc: "/apps/apple-calendar.svg",
```

Comment:
encouraging / before path file change here instead of before using it on every element that uses app.imgSrc